lib/src/parsers/contract-parser.ts

```diff
@@ -59,6 +60,13 @@ export function parseContract(
   const descriptionDoc = getJsDoc(klass);
   const description = descriptionDoc?.getDescription().trim();
 
+  // Handle Version
+  let version = "0.0.0";
+  const versionProp = getObjLiteralProp<ApiConfig>(decoratorConfig, "version");
+  if (versionProp) {
+    version = getPropValueAsStringOrThrow(versionProp)?.getLiteralText().trim();
+  }
```

Comment:
The contract definition specifies `version` as optional. In general we prefer letting the contract generators (OpenAPI 2 & 3) define the default values, rather than driving it with Spot's internal definitions. Something along the lines of:
```suggestion
  const versionProp = getObjLiteralProp<ApiConfig>(decoratorConfig, "version");
  const version = versionProp ? getPropValueAsStringOrThrow(versionProp).getLiteralText().trim() : undefined
```

Reply:
Excellent! Makes perfect sense to me. I will get that updated in the pull request as well as update the contract spec to reflect those changes. Thanks for the suggestion, and the rapid response!